src/main/java/codeu/controller/NotificationServlet.java

```diff
@@ -60,13 +60,14 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
 
         for(Event subscibedEvent:eventsLastSeen){
           String eventType = subscibedEvent.getType();
-          if(eventType.equals("Message")){
-            List<String> messageNotification= subscibedEvent.getInformation();
-            UUID conversationId = UUID.fromString(messageNotification.get(3));
+          List<String> information = subscibedEvent.getInformation();
+          if(eventType.equals("Message") && information.size() > 3){
+            String messageId = information.get(3);
+            UUID conversationId = UUID.fromString(messageId);
```

Comment:
I'm a little confused on why `conversationId` is based on `messageId`. Aren't they different?

Reply:
I just named it weirdly is still the id of the conversation, I just called the string messageId to differentiate it from the UUID conversationId. I can change the name.